ols/app/models/config.py

```diff
@@ -383,11 +380,59 @@ def validate_yaml(self) -> None:
         self.logging_config.validate_yaml()
 
 
+class DevConfig(BaseModel):
+    """Developer-mode-only configuration options."""
+
+    enable_dev_ui: bool = False
+    disable_question_validation: bool = False
+    llm_temperature_override: int | None = None
+
+    # TODO - wire this up once auth is implemented
+    disable_auth: bool = False
+
+    def __init__(self, data: Optional[dict] = None):
+        """Initialize developer configuration settings."""
+        super().__init__()
+        if data is None:
+            return
+        self.enable_dev_ui = str(data.get("enable_dev_ui", "False")).lower() == "true"
+        self.disable_question_validation = (
+            str(data.get("disable_question_validation", "False")).lower() == "true"
+        )
+        self.llm_temperature_override = data.get("llm_temperature_override", None)
```

Comment:
FWIW, here's a pattern I'm using for getting an int out of YAML: https://github.com/syedriko/lightspeed-service/blob/77970fd8a235dfa25e4ff1fd3d926214472463be/ols/app/models/config.py#L191

Reply:
this feels like a situation where it's 6 one way and half a dozen the other  :shrug: